spec/models/azure_engine_spec.rb

```diff
@@ -0,0 +1,181 @@
+require 'rails_helper'
+require 'spec_helper'
+require 'digest/md5'
+require 'tempfile'
+require 'securerandom'
+require 'tmpdir'
+
+module Azure
+  module Queue
+    class DummyQueueService
+      def create_message(queue_name, message)
+        nil
+      end
+      def list_messages(queue_name, timeout, options = {})
+        []
+      end
+      def create_queue(queue_name)
+        nil
+      end
+      def delete_message(queue_name, message_object_id, message_object_pop_receipt)
+        nil
+      end
+    end
+  end
+  module Entity
+    module Queue
+      class DummyMessage
+        def id
+          1
+        end
+        def pop_receipt
+          ''
+        end
+      end
+    end
+  end
+
+  module Blob
+    class DummyBlobObject
+      def name
+        'dummy'
+      end
+    end
+    class DummyBlobService
+      def create_block_blob(container, key, content)
+        nil
+      end
+      def list_blobs(container, options = {})
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'container' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UnusedParameters] has unused parameter 'options' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)